client/src/components/Modal/Modal.js

```diff
@@ -49,24 +40,15 @@ const Modal = ({
     <>
       {modalOpen ? (
         <ModalContainer modalOpen={modalOpen}>
```

Comment:
why giving the container the modalOpen property, it doesn´t do anything with it?
```suggestion
        <ModalContainer>
```